src/codegen/bitfield_unit.rs

```diff
@@ -27,7 +27,12 @@ where
         let byte_index = index / 8;
         let byte = self.storage.as_ref()[byte_index];
 
-        let bit_index = index % 8;
+        let mut bit_index = index % 8;
+        #[cfg(target_endian = "big")]
```

Comment:
It probably should use some runtime target information rather than build time config, because people may want to cross-compile.

Reply:
That's fine though, isn't it? This code is included in the target code.

Reply:
Oh, this is the code bindgen adds to the generated binding file. That makes sense then.